apps/src/studio/studio.js

```diff
@@ -234,6 +234,9 @@ function loadLevel() {
   }
   blocks.registerCustomGameLogic(Studio.customLogic);
 
+  Studio.legacyRuntime = Studio.customLogic;
```

Comment:
worth a comment explaining what it means to be a legacy runtime/why we still need to support it/if we have a plan to get rid of it entirely at some point?